src/server.rs

```diff
@@ -60,9 +60,16 @@ async fn create_server() -> axum::Server<AddrIncoming, IntoMakeService<axum::Rou
     server
 }
 
-fn setup_logging() {
+fn setup_logging(args: &cli::Args) {
+    let mut trace_level = tracing::Level::INFO;
```

Comment:
You can write this without mutable variables. Tip: `if` is an expression, not a statement.

Reply:
But from an API standpoint you are better of `impl`'ing it on the Cli type itself. 

Like https://github.com/DeterminateSystems/nix-installer/blob/329861feca73c00362bbe5004eb30fdb92ba398b/src/cli/arg/instrumentation.rs#L46-L53

Reply:
```
let trace_level = if verbose == 1 {
        tracing::Level::DEBUG
    } else if verbose == 2 {
        tracing::Level::TRACE
    } else {
        tracing::Level::INFO
    };
```